### PR TITLE
Milestone2: finish mobile version of the project

### PR DESCRIPTION
**Pull request by AHP15**
In this Milestone I added:
  - A work section with its styles and icons
  - An about section with its text, styles and icons
  - Changed the description of the README file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -56,6 +56,156 @@ <h1>
...
+                        <h4>Skills</h4>
+                        <ul class="ul-items">
+                            <li>Codekit</li>
+                            <li>GitHub</li>
+                            <li>Codepen</li>
+                            <li>Gitlab</li>
+                            <li>Terminal</li>
+                        </ul>
+                    </div>
+                </div>
```
Comment (Santiago220991) [inline index.html]: Kindly modify this according to the project requirements the technology requirements should be inside of a `ul`, and each of them in a `li`. :recycle: 
image

File: style.css [hunk 2]
```diff
@@ -80,8 +86,145 @@
...
+
+  .work-item {
+    background-image: url("./media/plaholder.png");
+    background-repeat: no-repeat;
+    border-radius: 8px;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    border: 1px solid lightgray;
+  }
```
Comment (Santiago220991) [inline style.css]: Please make sure that the background fit the container when you resize the screen. :eyes: 
image

**Review by Santiago220991 — CHANGES_REQUESTED** [review]
Hi @Abdessittir, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
:heavy_check_mark: Followed GitHub flow.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Good README file.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — APPROVED** [review]
## Hi @Abdessittir , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**